### PR TITLE
Update `readme` for w.org plugins page to match new copy

### PR DESCRIPTION
**Pull request by postphotos**
Updated readme to match new copy that more accurately describes Stream.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,24 +1,26 @@
 === Stream ===
-Contributors: lukecarbis, fjarrett, stream, xwp, kasparsd
+Contributors: xwp, kasparsd, derekherman, postphotos
```
Comment (derekherman) [inline readme.txt:2]: This should probably just be `xwp`. Not sure who has access to the stream account on wp.org and this list gives everyone write access to SVN. @kasparsd I can either give you the credentials to the `xwp` account so you can still push new versions of the plugin or keep you on the list but I don't see why I or Leo would be on this list. Thoughts?

Comment (postphotos) [reply readme.txt]: Agreed. Just pushed a change.

**Comment by postphotos** [comment]
Hey @derekherman - I've added the list of contributors as @kasparsd suggested and removed the other non-repo-committing folks from the list.

I think this is good to merge now.

File: readme.txt (hunk 1 above)
Comment (kasparsd) [reply readme.txt]: > Not sure who has access to the stream account on wp.org and this list gives everyone write access to SVN

@derekherman This contributors list is only included in the plugin credits and doesn't give SVN access. 

contributors

This attribution also makes the plugin show up on [user's list of contributions](https://profiles.wordpress.org/kasparsd/#content-plugins) which is nice:

contributions

The SVN access is managed via Committers under [the plugin settings](https://wordpress.org/plugins/stream/advanced/):

committers

**Pull request merged**